docs/secure_element.md

```diff
@@ -70,7 +70,8 @@ SDO implementation for ARM devices uses Microchip AT608A as the secure element.
    Note: only ECDSA is supported from SE.
    ```bash
    cd <c-code-sdk folder>
-   make CRYPTO_HW=true PK_ENC=ecdsa KEX=ecdh
+   cmake -DCRYPTO_HW=true -DPK_ENC=ecdsa -DKEX=ecdh .
```

Comment:
For uniformity, we might start the commands with $. 

Can be taken up at a later time.